src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -34,6 +34,12 @@ public class AgentKnobs
             new EnvironmentKnobSource("AGENT_SKIP_POST_EXECUTION_IF_CONTAINER_STOPPED"),
             new BuiltInDefaultKnobSource("false"));
 
+        public static readonly Knob SetMTUForContainerJobs = new Knob(
+            nameof(SetMTUForContainerJobs),
```

Comment:
Could you please rename it? This sounds more like a boolean flag rather than value parameter.

Reply:
Maybe MTUValueForContainerJobs?

Reply:
> Could you please rename it? This sounds more like a boolean flag rather than value parameter.

Maybe MTUValueForContainerJobs?